.github/workflows/dask.yml

```diff
@@ -0,0 +1,37 @@
+name: dask
+
+on:
+  schedule:
+    - cron: '0 15 * * *'
+  pull_request:
+    paths:
+      - 'dask/**'
+      - '.github/workflows/dask.yml'
+
+jobs:
+  examples:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10']
```

Comment:
having python 3.7 wouldn't be a problem for certain versions of dask?

Reply:
No it shouldn't be an issue. It will cause an older version of `distributed` to get pulled in, but https://github.com/optuna/optuna/pull/2023 doesn't use any new `distributed` features so is backwards compatible back to Python 3.7. Note that https://github.com/optuna/optuna/pull/2023 has Python 3.7 CI builds